aleth/main.cpp

```diff
@@ -202,6 +202,12 @@ int main(int argc, char** argv)
     fs::path configPath;
     string configJSON;
 
+    ///List_logging_channels
+    const std::vector <string> list_log_channels = {"block", "blockhdr", "bq", "chain", "client", 
```

Comment:
I see channels like `exec`, `vmtrace` missing.

Search for all uses of `Logger` to find them.

Reply:
Done. I checked all the instances of Logger

Reply:
@gumb0